tests/system/action/user/test_update.py

```diff
@@ -302,22 +273,46 @@ def test_update_permission_manage_user(self) -> None:
                 "default_number": "new default_number",
                 "default_structure_level": "new default_structure_level",
                 "default_vote_weight": "1.234000",
-                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_USERS,
-                "committee_ids": [60, 63],
             },
         )
 
-    def test_update_permission_user_can_manage(self) -> None:
-        """ May update group B and C fields """
+    def test_update_permission_group_A_no_permission(self) -> None:
```

Comment:
please add a short version of this for a user in committee scope and meeting scope.

Reply:
done